src/main/java/com/eprosima/microxrcedds/idl/templates/TypesHeader.stg

```diff
@@ -51,7 +51,7 @@ $definitions; separator="\n"$
 
 annotation(ctx, annotation) ::= <<>>
 const_decl(ctx, parent, const, const_type) ::= <<
-#define $const.name$ $const.value$
+static const $const.typeCode.cppTypename$ ardupilot_msgs_msg_$ctx.filename$_$const.name$ = $const.value$;
```

Comment:
```suggestion
static const $const.typeCode.cppTypename$ ardupilot_msgs_msg_$ctx.filename$_$const.name$ = $const.value$;
```
Can you avoid hard coding `ardupilot_msgs`? We generate messages from multiple packages.